Wallapop Architecture/Reducer.xctemplate/___FILEBASENAME___.swift

```diff
@@ -0,0 +1,35 @@
+import ComposableArchitecture
+
+/// sourcery: AutoMockable
+protocol ___FILEBASENAME___Delegate: AnyObject {
+    func present(_ state: ___FILEBASENAME___.State)
+}
+
+final class ___FILEBASENAME___: Reducer {
+    struct State {
+        
+    }
+
+    enum Actions: Action {
+        
+    }
+
+    weak var delegate: ___FILEBASENAME___Delegate?
+    private let effects: Effects
+
+    init(effects: Effects) {
+        self.effects = effects
+    }
+
+    func reduce(_ state: inout ___FILEBASENAME___.State, _ action: Actions) -> Effect<Actions> {
+        .none
+    }
+}
+
+extension ___FILEBASENAME___ {
+    struct Effects {
+        init() {
+            // Inject usecases
```

Comment:
What about doing this as a completion text?
```suggestion
            <#Inject usecases#>
```